frontend/lib/src/components/auth/auth_header.dart

```diff
@@ -20,6 +29,39 @@ class AuthHeader extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    const storage = FlutterSecureStorage();
+    final networkConfig = NetworkConfig();
+
+    void dropdownCallback(selectedValue) async {
+      if (selectedValue == 'logout') {
+        // note: csrf is for web, web implementation not thoroughly tested
+        await networkConfig.getCsrftoken();
+
+        final client = networkConfig.client;
+
+        final logoutResponse = await client.post(
+          logoutUrl,
+        );
+
+        if (logoutResponse.statusCode == HttpStatus.ok) {
+          final message = logoutResponse.data['data']['message'];
+
+          // delete the stored login key
+          await storage.delete(key: loginTokenKey);
+          Get.to(() => const LoginPage());
```

Comment:
get.to is incorrect here, triggering the back functionality on the phone after logging out goes back to user list page